app/src/main/java/com/mobiledevelopmentfoodapp/dto/cart

```diff
@@ -0,0 +1,11 @@
+package com.mobiledevelopmentfoodapp.dto
+
+//import androidx.room.Entity
+//import androidx.room.PrimaryKey
```

Comment:
Avoid pushing commented code to Version Control.  Version control has tools that makes this unnecessary.
- You can look at history to see what used to be there.
- You can use branches for experimental features.